### ISSUE TITLE
Legend of the chart

### ISSUE THREAD

**Issue by josojo** [gnosis/ido-ux#574]
Currently the legend looks quite profan:
![image.png](https://images.zenhubusercontent.com/5ddee9ea1d3e1600010ad960/30e85d22-fab4-4f7e-9af7-f00f63b6fdcb)

Can we please get a new design proposal + later on implementataion

**Comment by alongoni** [comment]
![image](https://user-images.githubusercontent.com/622217/121361356-9796d580-c90b-11eb-9c33-8710f9a97723.png)
First and simple approach: We can add a label for each legend.

Regarding #468 maybe we can add the current price in the legends. Also I can try to present this in the charts, but I'm not sure is the simple way to implement it.

Any suggestion? @josojo @cmagan @ramirotw

**Comment by josojo** [comment]
>Regarding #468 maybe we can add the current price in the legends.

yeah, if we find a good way to implement it, that would be great. Though in the past this chart library has not proven itself to be the most flexible and performant one.

For me, representing the price has no high prior, as it is visible within the auction dettails

**Comment by ramirotw** [comment]
Another option could be putting the current price value in the Current Price legend tooltip, that would probably be easier. I see value in having the chart's current price value as it can change depending on the Place Order's form.

**Comment by josojo** [comment]
The second option looks way more fancy and better. Though, I don't know how long it would take to implement. @maria-vslvn what do you think? Is it implementable?
The first option is very easy to implement...

**Comment by cmalfesi** [comment]
@maria-vslvn 
Please, check and let us know your thoughts to implement this. And estimate.

**Comment by cmagan** [comment]
I like the left one with the price on top of the white dashed line!

**Comment by elena-zh** [comment]
Hey @maria-vslvn , 
could you please also take a look at the #723 issue while implementing this task?
Thanks

**Comment by ramirotw** [comment]
@maria-vslvn lets implement the current price in the tooltip as adding the overlay to the graph could be complicated
